velox/connectors/hive/HiveDataSink.cpp

```diff
@@ -38,6 +38,46 @@ namespace facebook::velox::connector::hive {
 
 namespace {
 
+RowVectorPtr makeRowVector(
+    std::vector<std::string> childNames,
+    const std::vector<VectorPtr>& children,
+    int32_t numRows,
+    velox::memory::MemoryPool* pool) {
+  std::vector<std::shared_ptr<const Type>> childTypes;
+  childTypes.resize(children.size());
+  for (int i = 0; i < children.size(); i++) {
+    childTypes[i] = children[i]->type();
+  }
+  auto rowType = ROW(std::move(childNames), std::move(childTypes));
+  return std::make_shared<RowVector>(
+      pool, rowType, BufferPtr(nullptr), numRows, std::move(children));
+}
+
+RowVectorPtr filterPartitionColumns(
```

Comment:
Does it sound better to call this func makeDataInput() or filterOutPartitionColumns()?